test/jdk/java/foreign/TestLayouts.java

```diff
@@ -261,9 +267,9 @@ public void testPaddingIllegalByteSize() {
     @Test
     public void testStructToString() {
         StructLayout padding = MemoryLayout.structLayout(JAVA_INT).withName("struct");
-        assertEquals(padding.toString(), "[i4](struct)");
+        assertEquals(padding.toString().toLowerCase(), "[i4](struct)");
```

Comment:
I don't think we should ignore case here. Instead, I suggest we adjust the expecation according to whether the platform is big or little endian:
```java
        String basicLayoutString = (JAVA_INT.order() == ByteOrder.LITTLE_ENDIAN) ? "[i4](struct)" : "[I4](struct)";
        assertEquals(padding.toString(), basicLayoutString);
        var toStringUnaligned = padding.withByteAlignment(8).toString();
        assertEquals(toStringUnaligned, "8%" + basicLayoutString);
```

Reply:
That was my initial thought to distinguish the platforms in the test but I changed my mind as there is no need to check the capitalization for symbol. But I agreed to restore it back if we think it is necessary.

Reply:
Fixed as suggested above.